### PR TITLE
Added CSV without WARNINGS, they are treated as Errors

### PR DESCRIPTION
**Pull request by zvonkok**
Added correct CSV without WARNINGS, they are treated as Errors

### CODE REVIEW

**Comment by zvonkok** [comment]
@sjug @cep235 PTAL 

`operator-courier verify --ui_validate_io .` executes cleanly, without error or warning. 
ART builds are treating warnings as errors.

**Comment by zvonkok** [comment]
Updating the image-references as well, the operator references `:latest` so the image-reference has to reference `:latest` as well.

File: Makefile [hunk 1]
```diff
@@ -65,7 +65,7 @@ undeploy:
 	 ## Delete everything for the operator from the cluster
 #	-${TEMPLATE_CMD} $(DEPLOY_OBJECTS) $(DEPLOY_OPERATOR) $(DEPLOY_CRDS) $(DEPLOY_CRS) | kubectl delete -f -
 
-verify:	verify-gofmt
+verify:	verify-gofmt
```
Comment (sjug) [inline Makefile]: You don't need the trailing space here

File: manifests/olm-catalog/4.2/nfd.v4.2.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -1,19 +1,38 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
+  name: nfd.v4.2.0
+  namespace: placeholder
   annotations:
     capabilities: Basic Install
-    categories: "Configuration"
-    description: This software enables node feature discovery for Kubernetes. It detects hardware features available on each node in a Kubernetes cluster, and advertises those features using node labels.
+    categories: "Database"
```
Comment (sjug) [inline manifests/olm-catalog/4.2/nfd.v4.2.0.clusterserviceversion.yaml]: Where are these annotations used, and in what capacity?

File: manifests/olm-catalog/4.2/nfd.v4.2.0.clusterserviceversion.yaml [hunk 3]
```diff
@@ -1,19 +1,38 @@
...
+    
     provider: Red Hat
     support: Red Hat
     containerImage: 
     createdAt: 2019-05-30T00:00:00Z
     certified: "false"
     repository: https://github.com/openshift/cluster-nfd-operator
-  name: nfd.v4.2.0
-  namespace: placeholder
+    alm-examples: |-
```
Comment (sjug) [inline manifests/olm-catalog/4.2/nfd.v4.2.0.clusterserviceversion.yaml]: This was added to pass the OLM test?

**Comment by sjug** [comment]
Also isn't your second commit the same as https://github.com/openshift/cluster-nfd-operator/pull/26/?

File: manifests/olm-catalog/4.2/nfd.v4.2.0.clusterserviceversion.yaml (hunk 3 above)
Comment (zvonkok) [reply manifests/olm-catalog/4.2/nfd.v4.2.0.clusterserviceversion.yaml]: Yes, and all other changes were added to pass operator-courier tests.

File: Makefile (hunk 1 above)
Comment (zvonkok) [reply Makefile]:
```suggestion
verify:	verify-gofmt
```

File: manifests/olm-catalog/4.2/nfd.v4.2.0.clusterserviceversion.yaml (hunk 2 above)
Comment (zvonkok) [reply manifests/olm-catalog/4.2/nfd.v4.2.0.clusterserviceversion.yaml]: Just for filtering in the operatorhub ui.

**Comment by sjug** [comment]
/lgtm

Assuming that @zvonkok will squash the commits before merge

**Comment by zvonkok** [comment]
@sjug Here is the source-code for operator-courier https://github.com/operator-framework/operator-courier
Installing is done by: `pip3 install operator-courier` Version has to be `2.1.7`

**Comment by sjug** [comment]
@zvonkok Thanks!

**Pull request merged**